scripts/topology_generator/fat_tree.py

```diff
@@ -0,0 +1,125 @@
+import yaml
+
+def add_bidirectional_link(config, from_node, to_node, link_counter, preset = "default"):
+    config["links"][f"link{link_counter}"] = {
+        "from": from_node, "to": to_node, "preset-name": preset
+    }
+    link_counter += 1
+    config["links"][f"link{link_counter}"] = {
+        "from": to_node, "to": from_node, "preset-name": preset
+    }
+    return link_counter + 1
+
+def generate_fat_tree_config(k):
```

Comment:
I think name `k` is not really well
```suggestion
def generate_fat_tree_config(switch_ports_count):
```